docs/index.md

```diff
@@ -1596,26 +1596,25 @@ Environment variables and default values:
 | `--vmwarevsphere-compute-ip`      | `VSPHERE_COMPUTE_IP`      | -                        |
 
 ## Base Operating Systems
-The default base operating system for Machine is Boot2Docker on local providers
+The Machine provisioning system supports several base operating systems.
+The default base operating system is Boot2Docker on local providers
 (VirtualBox, Fusion, Hyper-V, etc) and the latest Ubuntu LTS supported
-by the cloud provider.  RedHat Enterprise Linux is also supported.  To use
-RHEL, you will need to select the image accordingly with the provider.  For
-example, in Amazon EC2, you could use a RedHat 7.1 AMI ("ami-12663b7a") as the
-`--amazonec2-ami` option which create an instance using RHEL 7.1 64-bit.
-
-## Release Notes
-
-### Version 0.2.0 (April 16, 2015)
-
-For complete information on this release, see the [0.2.0 Milestone project page](https://github.com/docker/machine/wiki/0.2.0-Milestone-Project-Page).
-In addition to bug fixes and refinements, this release adds the following:
-
-* Updated and refactored Driver interface For details, see
-[PR #694](https://github.com/docker/machine/pull/694).
-
-* Initial creation of an internal API, so Machine can be used as a library. For
-details, see [PR #553](https://github.com/docker/machine/issues/553).
-
-* Improvements and isolation of provisioning functionality, so Machine can
-provision and configure the Docker Engine based on OS detection. For details,
-see [PR #553](https://github.com/docker/machine/issues/553).
+by the cloud provider.
+
+| Operating System           | Version          | Notes                   |
+|----------------------------|------------------|-------------------------|
+| Boot2Docker                | 1.5+             | default for local       |
+| Ubuntu                     | 12.04+           | default for remote      |
+| Rancher                    | 0.3+             |                         |
+| Debian                     | 8.0+             | experimental            |
+| RedHat Enterprise Linux    | 7.0+             | experimental            |
+| CentOS                     | 7+               | experimental            |
+| Fedora                     | 21+              | experimental            |
+
+If you want to use a different base operating system on a remote provider,
+you will need to select the image accordingly for that provider.  For
+example, on DigitalOcean you would use the `--digitalocean-image` flag.
+For Amazon AWS, you would use the `--amazonec2-ami` flag.
+
+> Note: if you change the base image for a provider you may also need to change
```

Comment:
I'd include an example, e.g. you could use this one:

"as well.  For example, the default Red Hat AMI on EC2 expects the SSH user to be `ec2-user`, so you would have to specify this with `--amazonec2-ssh-user ec2-user`.